.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--- Add a pull request title above in this format -->
+<!--- real example: 'UIFR-222 Remove concept property setters from ObsResource classes' -->
+<!--- 'UIFR-JiraIssueNumber JiraIssueTitle' -->
+## Description of what I changed
+<!--- Describe your changes in detail -->
+<!--- It can simply be your commit message, which you must have -->
+
+
+## Issue I worked on
+<!--- This project only accepts pull requests related to open issues -->
+<!--- Want a new feature or change? Discuss it in an issue first -->
```

Comment:
Is this also required for the Java 17 compatibility?

Reply:
I removed this